### ISSUE TITLE
Linear gradients in SVG images from graphviz are not shown correctly

### ISSUE THREAD

**Issue by satirebird** [asciidoctor/asciidoctor-pdf#960]
Graphviz is able to generate linear gradients. The SVG image looks good but the pdf file doesn't show the gradient. The sources are attached. The SVG image was create with 'dot -T svg col_blue.gv -o col_blue.svg'

[gradient.zip](https://github.com/asciidoctor/asciidoctor-pdf/files/2584760/gradient.zip)

**Comment by satirebird** [comment]
After investigating the issue in detail I realized that it was an installation / configuration problem. I have two version of prawn-svg installed: 0.26.0 and 0.28.0. The gradient feature is was added starting from 0.27.0. The version constraint of asciidoctor-pdf alpha 16 is 
asciidoctor-pdf-1.5.0.alpha.16 depends on prawn-svg (< 0.28.0, >= 0.21.0)
In my setup asciidoctor-pdf uses prawn-svg 0.26.0 which dosn't have the gradient feature.
After installing prawn-svg 0.27.0 the gradients are shown as expected.
